packages/appkit/src/adapters/ChainAdapterBlueprint.ts

```diff
@@ -155,6 +161,8 @@ export abstract class AdapterBlueprint<
 
       return true
     })
+
+    this.emit('connectors', this.availableConnectors)
```

Comment:
I think it's fine since we already filter them [here](https://github.com/reown-com/appkit/blob/chore/adapter-improvement/packages/appkit/src/adapters/ChainAdapterBlueprint.ts#L154-L163) and it [ConnectorController](https://github.com/reown-com/appkit/blob/chore/adapter-improvement/packages/core/src/controllers/ConnectorController.ts#L35-L44) as well.